versions/1/input/changelog.spec.yml

```diff
@@ -0,0 +1,44 @@
+##
+## Describes the specification for the package's CHANGELOG file
+##
+spec:
```

Comment:
Is it not possible to `$ref` the whole spec from the integration one? Or are there differences between them?

Reply:
It worked! I referred to the changelog for integration.